src/Analyzers/MSTest.Analyzers/Resources.resx

```diff
@@ -117,6 +117,40 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="AssemblyInitializeShouldBeValidDescription" xml:space="preserve">
+    <value>Assembly Initialize attribute should follow the following layout to be valid:
+- it should be 'public' 
+- it should be 'static'
+- it should not be generic
+- it should take one parameter of type TestContext
+- return type should be 'void', 'Task' or 'ValueTask'
+- it should not be 'async void'
+- it should not be a special method (finalizer, operator...).</value>
+  </data>
+  <data name="AssemblyInitializeShouldBeValidMessageFormat_NotAsyncVoid" xml:space="preserve">
+    <value>Assembly Initialize '{0}' should return 'void', 'Task' or 'ValueTask'</value>
+  </data>
+  <data name="AssemblyInitializeShouldBeValidMessageFormat_NotGeneric" xml:space="preserve">
+    <value>Assembly Initialize '{0}' should not be generic</value>
+  </data>
+  <data name="AssemblyInitializeShouldBeValidMessageFormat_Ordinary" xml:space="preserve">
+    <value>Assembly Initialize '{0}' should be an 'ordinary' method</value>
```

Comment:
```suggestion
    <value>Assembly initialize method '{0}' should be an 'ordinary' method</value>
```

Reply:
I don't think we used '' around ordinary in other rules.

Reply:
@nohwnd we used '' it the other rules, does it better to remove it from them all?